data_dir/nemesis.yml

```diff
@@ -124,6 +124,10 @@
   - disruptive = True
   - kubernetes = True
   - free_tier_set = True
+- disrupt_mv_delete_range:
```

Comment:
Let's remove this new nemesis.
I think that the dtest of it is good enough to cover it and all the other changes here that increase the probability to hit these kind of issues randomly are good enough.

Reply:
Removed